docs/products/valkey/howto/create-valkey-read-replica.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Create read replica in Aiven for Valkey
+sidebar_label: Create read replica
+early: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+[Aiven for Valkey read replica](/docs/products/valkey/concepts/read-replica) enables data replication from a primary to a replica service, improving performance and increasing redundancy for high availability and disaster recovery.
+
+## Prerequisites
+
+- An Aiven account and API token.
+- [Aiven CLI tool](https://github.com/aiven/aiven-client).
+
+## Limitations
+
+- You can create a maximum of **5 read replicas** per primary service.
+- Read replicas are supported only on the **Startup** plan.
+
+## Create a read replica
+
+<Tabs>
+<TabItem value="1" label="Aiven Console" groupId="group1" default>
```

Comment:
The `groupId` should be on `Tabs` the element :)

Reply:
Fixed